VALUES.md

```diff
@@ -0,0 +1,30 @@
+# Ayo.js Community Values
+
+We don't care how you pronounce it: be it "ey-yo", "I-oh", or "Awooooo" -- the Ayo.js project strives to create an environment where you can feel at home and focus on your contributions to this nice shared space that we all benefit from.
+
+Ayo.js is about the humans that help make it happen. It's a project aimed at creating a new foundation of project governance and management that brings **humans** and their concerns front and center.
+
+## Ayo.js is more than the code in its repository.
+
+Documentation, long-term support, translations and localizations, package management, libuv, v8 and Chakra, builds/ops, testing and its infrastructure, hardware and IoT, Android, streams, community events, facilitation, moderation and education. These and more are all concerns that can be handled separately and by members who might be much better qualified for their individual tasks than a central decision body.
```

Comment:
I think there is an obvious "greater than the sum of its parts" point hiding in here that would be cool to make more explicit :smile: